### ISSUE TITLE
Ground Suggestions in Citations or Evidence

### ISSUE THREAD

**Issue by CaptainEmerson** [barik/survey-chp-ds4se#13]
this sometimes results in
some suggestions/indications for better survey design that are not well
grounded and motivated.

**Comment by CaptainEmerson** [comment]
"How did we know which statements to ask? Our statements came directly from
the stories told by participants during our semi-structured interviews." This
seems a good idea, but relying only on interviews for designing survey
questions may lead to other biases (e.g., what happens if all the interviewed
people are not a representative sample of important matters that should also be
investigated in the survey). I think a good survey has to be also grounded on a
through literature review. I'd encourage you to, at least, mention it.

"We recommend that the survey not take more than 15 minutes." That's a
sensible advice, can you also mention on what you ground it on?

"providing a strong initial recruitment letter is perhaps just as important as
the survey" From my experience, most people do not read this kind of
introductions. What is your statement based on? Then, I would also mention
something about how to write such a letter.

In short, I would encourage to point the reader to why you suggest these (and
other) things.

**Comment by CaptainEmerson** [comment]
Could just be "in our experience" or "based on interviews with other researchers"

**Comment by CaptainEmerson** [comment]
"A final reason is that surveys give us a certain sense of quantitative comfort
because the survey responses can be downloaded into a spreadsheet or other
statistical analysis software. A popular form of question-style, Likert
statements (Strongly Disagree to Strongly Agree), can then be quantitatively
analyzed through various descriptive and statistical techniques that are
already familiar to us."
In this case, I would shorten this part and not talk that much about "comfort"
and "familiar to us", because that may be definitely not true (probably, some
types of researchers are more comfortable with other types of analyses). I'd
just mention that they allow to draw statistical conclusions from the
aggregated answers.

"these lures are only surface benefits." Well, no :) These are real
benefits, the point is that the survey has to be carefully designed to get them
right.
